R/fortify-igraph.R

```diff
@@ -1,29 +1,13 @@
 #' Fortify method for networks of class \code{\link[igraph:igraph-package]{igraph}}
 #'
-#' @param model an object of class \code{\link[igraph:igraph-package]{igraph}}
+#' @param model an object of class \code{\link[igraph:igraph-package]{igraph}}.
 #' @param data not used by this method.
 #' @param layout a function call to an
 #'   \code{\link[igraph:igraph-package]{igraph}} layout function, such as
 #'   \code{\link[igraph]{layout_nicely}} (the default), or a 2 column matrix
 #'   giving the x and y coordinates for the vertices.
 #'   See \code{\link[igraph]{layout_}} for details.
-#' @param arrow.gap a parameter that will shorten the network edges in order to
-#'   avoid overplotting edge arrows and nodes; defaults to \code{0} when the
-#'   network is undirected (no edge shortening), or to \code{0.025} when the
-#'   network is directed. Small values near \code{0.025} will generally achieve
-#'   good results when the size of the nodes is reasonably small.
-#' @param by a character vector that matches an edge attribute, which will be
-#'   used to generate a data frame that can be plotted with
-#'   \code{\link[ggplot2]{facet_wrap}} or \code{\link[ggplot2]{facet_grid}}. The
-#'   nodes of the network will appear in all facets, at the same coordinates.
-#'   Defaults to \code{NULL} (no faceting).
-#' @param scale whether to (re)scale the layout coordinates. Defaults to
-#'   \code{TRUE}, but should be set to \code{FALSE} if \code{layout} contains
-#'   meaningful spatial coordinates, such as latitude and longitude.
-#' @param stringsAsFactors whether vertex and edge attributes should be
-#'   converted to factors if they are of class \code{character}. Defaults to
-#'   the value of \code{getOption("stringsAsFactors")}, which is \code{TRUE} by
-#'   default: see \code{\link[base]{data.frame}}.
+#' @inheritParams format_fortify
```

Comment:
Was working on something similar offline, but your code is much cleaner than what I came up with!

Reply:
Thanks^^